src/styles/card.css

```diff
@@ -0,0 +1,105 @@
+/*  CARD BLOCK:  */
+
+.card {
+  padding: 32px 0 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: 198px;
+}
+
+.card__info {
+  display: flex;
+  flex-direction: column;
+  margin: 0 16px;
+  --dark-color: #060b35;
+  --grey-color: #616070;
+}
+
+.card__info-logo {
+  width: 160px;
+  height: 134px;
+  margin: 0 auto 40px;
+}
+
+.card__info-title {
+  color: var(--dark-color);
+  font-family: inherit;
+  font-weight: 500;
+  font-size: 12px;
+
+  line-height: 18px;
+  margin: 0 0 4px;
```

Comment:
As for me, it's better to reset default margins somewhere in `reset` file and set margins only for that sides that you need (it's more readable)

Reply:
But it's not critical here